parm/config/gfs/config.ufs

```diff
@@ -151,7 +151,7 @@ case "${fv3_res}" in
         export layout_y=8
         export layout_x_gfs=8
         export layout_y_gfs=8
-        export nthreads_fv3=1
+        export nthreads_fv3=2
```

Comment:
This resolution should be tested more broadly with a S2SW forecast only application.